docs/pages/common_issues/isr.mdx

```diff
@@ -30,7 +30,7 @@ import { CloudFrontClient, CreateInvalidationCommand } from "@aws-sdk/client-clo
 
 const cloudFront = new CloudFrontClient({});
 
-async function invalidateCFPaths(paths: string[]) {
+async function invalidateCloudFrontPaths(paths: string[]) {
```

Comment:
Can you also update the reference to the function below:
```
// This costs $0.005 x 3 = $0.015 after the first 1000 paths
await invalidateCFPaths(["/page/a", "/page/b", "/page/c"]);
// This costs $0.005, but also invalidates other routes such as "page/d"
await invalidateCFPaths(["/page/*"]);
```

Reply:
Yup will do!